titlecheck.lic

```diff
@@ -0,0 +1,66 @@
+=begin
+This script requires that you have ;dependency installed... it will not work otherwise
+Checks for new titles since the last time you ran ;titlecheck.
+Usage: ;titlecheck <title set>  (e.g. ;titlecheck moonmage)
+
+author: Crannach
+game: dr
+=end
+
+class TitleCheck
+  include DRC
```

Comment:
❓ Does this need `custom_require([common])`?

Reply:
yep for the bput